RocketChat/Rocket.Chat
# chore: added_url_shortner_for_messages with long urls

### Proposed changes 
I have implemented a URL shortening feature for long URLs sent in messages. This change is intended to improve the readability of messages containing lengthy URLs.

### Code Changes
I have added a new function called ShortURLInMessage in the file ShortURLInMessage.ts. This function takes a Markdown (md) structure and truncates URLs within LINK elements to a maximum of 100 characters, appending ellipses (...) if the URL exceeds this length.

## Issue(s)

Closes RocketChat/feature-requests#881

## Steps to test or reproduce
1. Send a message containing a long URL in any channel or group.
2 .Observe the rendered message to ensure that the URL is truncated if it exceeds 100 characters.

<img width="1709" alt="Screenshot 2024-01-28 at 7 17 49 PM" src="https://github.com/RocketChat/Rocket.Chat/assets/125182653/b8f88499-d3f7-4053-a875-d003d506b8b5">
